modules/apps/document-library/document-library-document-conversion/bnd.bnd

```diff
@@ -15,6 +14,7 @@ Import-Package:\
 Require-Capability:\
 	osgi.extender;\
 		filter := "(&(osgi.extender=osgi.configurator)(version>=1.0)(!(version>=2.0)))"
+-fixupmessages: "^Classes found in the wrong directory: \\\\{META-INF/versions/9/";is:=warning
```

Comment:
@jeffwu0724 This seems weird, you only changed the compileInclude to compileOnly, why you need to add this? Removing a third lib requires more build requirements does not make sense.